tests/migration/migration.go

```diff
@@ -2939,6 +2942,62 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 		})
 	})
 
+	Context("[Serial]Cluster-Wide CPU Feature Disablement using cpu-model-to-features-to-disable ConfigMap", Serial, func() {
+		const cpuFeaturesToDisable = "aes"
+		var cpuModelsThatRequireAES = []string{"Broadwell-IBRS", "Broadwell-noTSX-IBRS", "Broadwell-noTSX", "Broadwell",
+			"Cascadelake-Server-noTSX", "Cascadelake-Server", "Cooperlake", "EPYC-IBPB", "EPYC-Milan", "EPYC-Rome",
+			"EPYC", "Haswell-IBRS", "Haswell-noTSX-IBRS", "Haswell-noTSX", "Haswell", "Icelake-Client-noTSX",
+			"Icelake-Client", "Icelake-Server-noTSX", "Icelake-Server", "IvyBridge-IBRS", "IvyBridge", "Opteron_G4",
+			"Opteron_G5", "SandyBridge-IBRS", "SandyBridge", "Skylake-Client-IBRS", "Skylake-Client-noTSX-IBRS",
+			"Skylake-Client", "Skylake-Server-IBRS", "Skylake-Server-noTSX-IBRS", "Skylake-Server", "Snowridge",
+			"Westmere-IBRS", "Westmere",
+		}
+		var chosenModel string
+		BeforeEach(func() {
+			nodes := libnode.GetAllSchedulableNodes(virtClient)
+			for _, model := range cpuModelsThatRequireAES {
+				if labelExistsOnAtLeastTwoNodes(nodes.Items, v1.CPUModelLabel+model) {
+					cm := &k8sv1.ConfigMap{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: util2.CpuModelDisableFeaturesConfigMapName,
+						},
+						Data: map[string]string{model: cpuFeaturesToDisable},
+					}
```

Comment:
I did just a quick look through the PR, but according to the description, the config map should be created in the KubeVirt install namespace. If this test passes with the default namespace, probably there is a bug somewhere?

Reply:
ouch, right. I must have missed that.